app/helpers/procedure_helper.rb

```diff
@@ -10,11 +10,9 @@ def procedure_lien(procedure)
   end
 
   def procedure_libelle(procedure)
-    parts = [procedure.libelle]
-    if procedure.brouillon?
-      parts << '(brouillon)'
-    end
-    parts.join(' ')
+    parts = procedure.brouillon? ? [content_tag(:span, 'démarche en test', class: 'badge')] : []
```

Comment:
👍 pour le badge, par contre ça te va si on reste sur juste "brouillon" (car on ne parle pas encore de "test" dans l'interface admin il me semble), et en suffixe ? Mais après réflexion, si tu mets "test", je suppose que c'est car si tu créés un brouillon sur une démarche en brouillon, ça fait [2 fois brouillon](https://youtu.be/feHdY7KZP_M?t=3)...

Quid de "démarche non publiée" ?